test.js

```diff
@@ -0,0 +1,64 @@
+var fs = require('fs');
+var crypto = require('crypto');
+
+var filename = 'README.md';
+var entryRegex = /\*\s[a-zA-Z\s]*\s\[.*\]\(.*\)/g;
```

Comment:
Good catch, @abinashmeher999 . I was thinking of greedy matches as well, but I didn't think of that particular one, so went for the easier `.*` syntax :stuck_out_tongue: Thanks!